tests/cases/fourslash/goToDefinition_filteringMappedType.ts

```diff
@@ -0,0 +1,7 @@
+///<reference path="fourslash.ts"/>
+
+////const obj = { /*def*/a: 1, b: 2 };
+////const filtered: { [P in keyof typeof obj as P extends 'b' ? never : P]: 0; } = { a: 0 };
+////filtered.[|/*ref*/a|];
+
+verify.goToDefinition("ref", "def");
```

Comment:
Hm, despite the fact that this test passes after my change... I can't make it work in the IDE. Note that I'm definitely using the local version. I've both changed my settings to use this
```json
{
    "typescript.tsdk": "built/local"
}
```

and I made sure that it's actually being loaded by restarting the TS server and observing the TypeScript log output. I can see:
> Using tsserver from: ~/Code/TypeScript/built/local/tsserver.js

Any ideas on what I could do wrong here?